src/agent/src/agent.cpp

```diff
@@ -164,6 +165,11 @@ void Agent::Run()
                         },
                         m_messageQueue);
                 }
+                else if (cmd.Module == "restart")
```

Comment:
After rebase, add module name to [here](https://github.com/wazuh/wazuh-agent/blob/master/src/agent/command_handler/include/command_entry/command_entry.hpp#L17).